scripts/windows/test_task.bat

```diff
@@ -89,16 +89,16 @@ rem work-around for failing to dl(mysqli) with OPcache (https://github.com/php/p
 if "%OPCACHE%" equ "1" set OPCACHE_OPTS=%OPCACHE_OPTS% -d extension=mysqli
 
 rem prepare for enchant
-mkdir %~d0\usr\local\lib\enchant-2
+if NOT EXIST %SCRIPT_DRIVE%\usr\local\lib\enchant-2 mkdir %SCRIPT_DRIVE%\usr\local\lib\enchant-2
```

Comment:
I'd like to call out that these also would not affect a clean-room environment, such as GitHub actions; and are not that readable to most competent engineers, because unlike Microsoft developers; in 30 years (probably took a week of frustration), posix devs created short-syntax `mkdir -p`.